model/infinispan/src/main/java/org/keycloak/connections/infinispan/InfinispanConnectionProvider.java

```diff
@@ -27,7 +27,7 @@
 
     String REALM_CACHE_NAME = "realms";
     String REALM_REVISIONS_CACHE_NAME = "realmRevisions";
-    int REALM_REVISIONS_CACHE_DEFAULT_MAX = 10000;
+    int REALM_REVISIONS_CACHE_DEFAULT_MAX = 20000;
```

Comment:
Upped this default value to help with testing with large number of realms. Should be covered anyway by a realms cache size in standalone.xml